.github/workflows/github-deploy.yml

```diff
@@ -59,12 +59,13 @@ jobs:
     # alternatively, to publish when a GitHub Release is created, use the following rule:
     # if: github.event_name == 'release' && github.event.action == 'published'
     steps:
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@v4
         with:
-          name: artifact
+          pattern: cibw-*
           path: dist
+          merge-multiple: true
 
-      - uses: pypa/gh-action-pypi-publish@v1.4.2
+      - uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: __token__
```

Comment:
I suggest switching to[ Trusted Publishing](https://github.com/marketplace/actions/pypi-publish#trusted-publishing), which doesn't require a username and password.